packages/cli/generators/relation/relationBelongsTo.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const ast = require('ts-simple-ast');
+const path = require('path');
+const RelationGenerator = require('./relation');
+const utils = require('../../lib/utils');
+const relationUtils = require('./relationutils');
+
+const CONTROLLER_TEMPLATE_PATH_BELONGS_TO =
+  'controller-relation-template-belongs-to.ts.ejs';
+
+module.exports = class RelationBelongsTo extends RelationGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  generateControllers(options) {
+    this.artifactInfo.sourceModelClassName = options.sourceModel;
+    this.artifactInfo.targetModelClassName = options.destinationModel;
+    this.artifactInfo.paramTargetModel = utils.camelCase(
+      options.destinationModel,
+    );
+    this.artifactInfo.sourceRepositoryClassName =
+      this.artifactInfo.sourceModelClassName + 'Repository';
```

Comment:
wouldn't be better to allow the user to select a list of available repositories for that model?. Or perhaps this could be the default option  unless you specify `--list-repositories`  ?.

Reply:
@marioestradarosa  @bajtos  @raymondfeng 

Ok. We will add to allow the user to select a list of available repositories for that model. Please continue review the code.